README.md

```diff
@@ -89,4 +89,8 @@ to build in prod use command
 
  
 Website address https://kp01012022.azurewebsites.net/
- 
\ No newline at end of file
+ 
+ Module 11-
+ everytime you reload gitpod you need to go to github > settings > developer settings and into your oauth and update the url and callback url with the new numbers on gitpod
+
+ Before merging, a new OAUTH app in github and set env variables in azure
```

Comment:
Double check the wording here. It's not quite right

Reply:
It would be useful to have this expanded on, so that someone who is new to this project understands the steps involved